### PR TITLE
Add functionality to pet index

### PR DESCRIPTION
**Pull request by jpc20**
Adds ability to update and delete a pet from the pets index and the shelter pets index

### CODE REVIEW

File: app/views/pets/edit.html.erb [hunk 1]
```diff
@@ -12,5 +12,5 @@
   <input type="text" name="pet[approximate_age]" id="approximate_age" value=" <%= @pet.approximate_age%>"><br>
   <label for="sex">Sex: </label>
   <input type="text" name="pet[sex]" id="sex" value=" <%= @pet.sex %>"><br>
-  <input type="submit" id="Submit">Update Pet</input>
+  <input type="submit" id="Submit" value="Update Pet">
```
Comment (jpc20) [inline app/views/pets/edit.html.erb:15]: Fix mistake where I had Update Pet as plain text next to the link

File: app/views/pets/index.html.erb [hunk 2]
```diff
@@ -6,5 +6,7 @@
     <li>Approximate age: <%= pet.approximate_age %></li>
     <li>Sex: <%= pet.sex %></li>
     <li>Shelter: <%= pet.shelter.name %></li>
+    <li><%= link_to "Update Pet", "/pets/#{pet.id}/edit", id: "update-pet-#{pet.id}"%></li>
+    <li><%= link_to "Delete Pet", "/pets/#{pet.id}", id: "delete-pet-#{pet.id}", method: :delete %></li>
```
Comment (jpc20) [inline app/views/pets/index.html.erb:10]: This is a much better wat to make a link. I am hoping to have time to replace the other ones.

File: spec/features/pets/user_can_delete_a_pet_spec.rb [hunk 3]
```diff
@@ -20,4 +20,18 @@
...
+    click_link "delete-pet-#{@pet.id}"
+    expect(current_path).to eq("/pets")
+    expect(page).to_not have_content(@pet.name)
+  end
+
+  it "user can delete a pet from the shelter pets index" do
+    visit "/shelters/#{@shelter.id}/pets"
+    click_link "delete-pet-#{@pet.id}"
+    expect(current_path).to eq("/pets")
+    expect(page).to_not have_content(@pet.name)
```
Comment (jpc20) [inline spec/features/pets/user_can_delete_a_pet_spec.rb:34]: I would like to make these tests more robust to test for edge cases.
